src/main/java/frc/robot/Robot.java

```diff
@@ -481,6 +487,18 @@ public void testPeriodic() {}
   @Override
   public void testExit() {}
 
+  public void checkAlliance() {
+    Alliance checkedAlliance = DriverStation.getAlliance();
+
+    if (DriverStation.isDSAttached() && checkedAlliance != currentAlliance) {
+      currentAlliance = checkedAlliance;
+
+      goClosestGrid = new GoClosestGrid();
+      autoChooser.addDefaultOption("TwoBridgeOver", new TwoCargoOver());
+      autoChooser.addOption("TwoBridgeUnder", new TwoCargoUnder());
```

Comment:
This is the same code as lines 153-154, can we move these into a `buildAutoChooser()` method?

Reply:
good point, ill do that now